go.mod

```diff
@@ -1,7 +1,7 @@
 module github.com/ChainSafe/gossamer
 
 require (
-	github.com/ChainSafe/chaindb v0.1.5-0.20210112170640-08c8f45c6f42
+	github.com/ChainSafe/chaindb v0.1.5-0.20210113205339-507b3a78c512
```

Comment:
Should we make a tag on chaindb repo for this so we can require a specific release (instead of master)?

Reply:
hmm good idea, should I tag 0.1.0 at this commit?

Reply:
Not sure, what's the difference between releases and tags?  I see we're at release 0.0.1, but the tags are at v0.1.4.  Maybe 0.1.5 or 0.2.0, are these updates breaking changes?